### DISCUSSION TITLE
Need help in the restoration of repository blueedgetechno/windows11

### DISCUSSION THREAD

**Discussion by blueedgetechno** [community/community#22435] [New to GitHub]
<p>My repo blueedgetechno/windows11 has been disabled by github for trademark violation.</p>
<p>I already opened two support tickets but both of them has been closed by github. I want to reappeal so that I can make the required changes to remove the trademark and also change the name of the repo.</p>
<p>I’m afraid if I will open a new ticket they might take action against by github account. What should I do?</p>

**Comment by tuves** [comment, marked as answer]
<p>Hi <a class="mention" href="/blueedgetechno">@blueedgetechno</a>-</p>
<p>I just looked up your support ticket and it looks like you’re already in communication with our team. I know there may sometimes be delays but trust that if you have an open ticket with Support, they will get to it. Opening a 2nd ticket, unfortunately, will only slow things down. Good luck!</p>

**Comment by blueedgetechno** [comment]
<p>Thanks tuves, It certainly helped me.</p>

**Comment by metropo228** [comment]
<aside class="quote" data-post="1" data-topic="207595">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/need-help-in-the-restoration-of-repository-blueedgetechno-windows11/207595">Need help in the restoration of repository blueedgetechno/windows11</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    My repo blueedgetechno/windows11 has been disabled by github for trademark violation. 
I already opened two support tickets but both of them has been closed by github. I want to reappeal so that I can make the required changes to remove the trademark and also change the name of the repo. 
I’m afraid if I will open a new ticket they might take action against by github account. What should I do?
  </blockquote>
</aside>
